src/providers/authentication/authentication.ts

```diff
@@ -30,6 +32,8 @@ export class AuthenticationProvider {
   public initialiseAuthentication = (): void => {
     this.authenticationSettings = this.appConfig.getAppConfig().authentication;
     this.employeeIdKey = this.appConfig.getAppConfig().authentication.employeeIdKey;
+    // TODO - get the name key correctly (will also vary between dev and production)
```

Comment:
Can this TODO be solved in this PR?

Reply:
Can only resolve this when I know where the property is and the config service has been amended to include it.